contracts/interfaces/IWETH.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.5.0;
+
+interface IWETH {
```

Comment:
should use https://unpkg.com/@uniswap/v3-periphery@1.3.0/contracts/interfaces/external/IWETH9.sol